numaprom/clients/prometheus.py

```diff
@@ -52,7 +52,7 @@ def query_metric(
             df = pd.concat(frames, ignore_index=True)
             df.sort_values(by=["timestamp"], inplace=True)
             df["timestamp"] = pd.to_datetime(df["timestamp"], unit="s")
-            df.reset_index(drop=True, inplace=True)
+
```

Comment:
Can you raise a custom exception, e.g. `InvalidDatashapeError`, if the data is not of the correct shape?

Reply:
Added it in training